src/components/trade/LastTrades/LastTradesWidget.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { TokenDetails } from 'types'
+import { LastTrades, LastTradesItem } from './LastTrades'
+
+import { useQuery, gql } from '@apollo/client'
+import BigNumber from 'bignumber.js'
+import { calculatePrice } from '@gnosis.pm/dex-js'
+
+const GET_TRADES = gql`
+  query GetTrades {
```

Comment:
I wonder if `subscription` will work. With WebSockets

Reply:
I think so, I haven't got to try it, but I understand it's transparent and that's the magic of using their `useQuery`. Anyways, I just kept things simple in this PR as explained in the description